cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -328,18 +325,16 @@ void addApplication(Id.Application id,
   /**
    * Deletes a schedules from a particular program
    * @param program defines program from which a schedule is being deleted
-   * @param programType defines the type of the program
    * @param scheduleName the name of the schedule to be removed from the program
    */
-  void deleteSchedule(Id.Program program, SchedulableProgramType programType, String scheduleName);
```

Comment:
Not all `ProgramType`s can be passed to scheduler APIs, and by `SchedulableProgramType` being a subset of `ProgramType`, this helps restrict which `ProgramType`s are allowed to be scheduled/unscheduled.